### DISCUSSION TITLE
Vaultwarden is not updated by Watchtower - command problem?

### DISCUSSION THREAD

**Discussion by Sjonnie2018** [containrrr/watchtower#1255] [Q&A] [answered]
Hi All,

I have installed Vaultwarden a few months ago using this guide: https://pimylifeup.com/raspberry-pi-bitwarden/

I thought I could use Watchtower to update the image. But after installing and checking the image for the last hour or so, I see no change in the Vaultwarden image. I tried searching the issue list over here and did some research but I "fear" I am at a loss.

When I use "docker ps" this is what I see:

```
docker ps
CONTAINER ID   IMAGE                                COMMAND                  CREATED             STATUS                PORTS                                                                                                                             NAMES
0cee4f048153   containrrr/watchtower:armhf-latest   "/watchtower"            51 minutes ago      Up 51 minutes         8080/tcp                                                                                                                          watchtower
dbe02448e523   portainer/portainer-ce:2.11.1        "/portainer"             About an hour ago   Up About an hour      0.0.0.0:8000->8000/tcp, :::8000->8000/tcp, 0.0.0.0:9000->9000/tcp, :::9000->9000/tcp, 0.0.0.0:9443->9443/tcp, :::9443->9443/tcp   portainer
226c1221e266   vaultwarden/server:latest            "/usr/bin/dumb-init …"   3 months ago        Up 3 days (healthy)   127.0.0.1:3012->3012/tcp, 127.0.0.1:8080->80/tcp                                                                                  Bitwarden
```
The only difference I see is the command string which looks a bit out of the ordinary.

I am running docker on an RPi 3B with Bullseye.

Hope you are able to shed some light on this :-)

**Comment by piksel** [comment, marked as answer]
The image was latest changed over 2 months ago: https://hub.docker.com/layers/vaultwarden/server/latest/images/sha256-c2e43b5590b35b348773e7a2d56ea3eb1dd0107ccb642fba30b213120a63ded6?context=explore

You can check the watchtower logs by doing 
```
docker logs watchtower
```
to see what it's doing.

**Comment by Sjonnie2018** [comment]
Hi @piksel,

Thanks for your reply!

I checked portainer again and discovered that Vaultwarden was updated this afternoon. That's about 24 hours after installing Watchtower. Which is fine because it confirms Watchtower is working. And the log (see below) says as much ;-)

I ran the command prior to checking and this was reported:
```
docker logs watchtower
time="2022-03-20T15:01:10Z" level=info msg="Watchtower 1.4.0"
time="2022-03-20T15:01:10Z" level=info msg="Using no notifications"
time="2022-03-20T15:01:10Z" level=info msg="Checking all containers (except explicitly disabled with label)"
time="2022-03-20T15:01:10Z" level=info msg="Scheduling first run: 2022-03-21 15:01:10 +0000 UTC"
time="2022-03-20T15:01:10Z" level=info msg="Note that the first check will be performed in 23 hours, 59 minutes, 59 seconds"
time="2022-03-21T15:01:53Z" level=info msg="Found new vaultwarden/server:latest image (86445ffb6c1f)"
time="2022-03-21T15:01:53Z" level=info msg="Stopping /Bitwarden (226c1221e266) with SIGTERM"
time="2022-03-21T15:01:56Z" level=info msg="Creating /Bitwarden"
time="2022-03-21T15:01:58Z" level=info msg="Session done" Failed=0 Scanned=3 Updated=1 notify=no
```
So all is good! I was a bit impatient as it turns out :-)
